components/service-binding-usage-controller/docs/architecture.md

```diff
@@ -1,44 +1,45 @@
 # Architecture
 
-The Binding Usage Controller workflow steps are as follows:
+The ServiceBindingUsage Controller workflow looks as follows:
 
-1. The Binding Usage Controller watches the ServiceBindingUsages in all Namespaces and triggers the logic described in this section. The user decides on the names of the ServiceBindingUsages.
+![Architecture](assets/architecture.svg)
 
-2. The Binding Usage Controller fetches all labels from the bindingLabels section in the ClusterServiceClass for which the ServiceBinding was created.
+1. The ServiceBindingUsage Controller watches ServiceBindingUsages in all Namespaces and triggers the logic described in this section. The user decides on the names of ServiceBindingUsages.
 
-3. The Binding Usage Controller creates a PodPreset which is applied to all Pods with the **use-{serviceBindingUsageUID}** label set to `{serviceBindingUsage resource version}`. The name of the PodPreset is the SHA1-encoded name of the ServiceBindingUsage. The PodPreset injects a Secret with the same name as specified in the  **serviceBindingRef.name** field into those Pods. By default, the prefixing of the injected environment variables is disabled. Set the **envPrefix.name** to enable the prefix. **envPrefix.name** is used to prefix the name of the environment variable.
+2. The ServiceBindingUsage Controller fetches all labels from the **bindingLabels** section in the ClusterServiceClass for which the ServiceBinding was created.
 
-4. The Binding Usage Controller finds the corresponding UsageKind specified by the **usedBy.Kind** field. The UsageKind defines the resource to be labelled and the field which contains labels.
+3. The ServiceBindingUsage Controller finds the corresponding UsageKind specified in the **usedBy.Kind** field. The UsageKind defines the resource to be labelled and the field which contains labels.
```

Comment:
```suggestion
3. The ServiceBindingUsage Controller finds the corresponding UsageKind specified in the **usedBy.Kind** field. The UsageKind defines the resource to be labeled and the field which contains labels.
```
Please use the AmE spelling (see the corresponding [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology)).